.github/workflows/build_rpms.yml

```diff
@@ -0,0 +1,39 @@
+name: RPM Build
+
+on: [pull_request, push]
+
+jobs:
+  rpm_build:
+    name: "Build RPMs"
+    runs-on: ubuntu-latest
+    container: registry.gitlab.com/majorhayden/containers/rpm-builder:latest
```

Comment:
What is the story here? Do you maintain these, or do we move to a "osbuild" account for the registry in the future? Or is this just for now and we look for other solutions when this breaks?

Reply:
I was mainly just trying to speed things up here by having a pre-prepared container. We could use the default Fedora 31 container instead.